script.js

```diff
@@ -0,0 +1,12 @@
+function closeMenu() {
+  document.querySelector('.mobile-nav-menu').style.display = 'none';
+  document.body.style.overflow = 'scroll';
+}
+
+closeMenu();
+
+const union = document.querySelector('#union');
+union.addEventListener('click', () => {
+  document.querySelector('.mobile-nav-menu').style.display = 'block';
+  document.body.style.overflow = 'hidden';
```

Comment:
- [ ] Kindly ensure that you improve your design in the way it sticks as much as possible to the Figma template design(e.g.: Improve the blur effect on the background, fonts on text(weight,...), padding, margin,...)

| Figma Design | Your Design |
| -------------------- | ---------------- |
| ![Screenshot from 2023-01-04 13-57-26](https://user-images.githubusercontent.com/83042742/223242457-5b5a8c30-dd8e-4923-878d-4ab4eadb7d13.png) | ![Screenshot from 2023-03-06 23-50-08](https://user-images.githubusercontent.com/83042742/223242768-6b1f8246-2c9e-4ee8-9738-2c101d808702.png) |